crates/sui-axelar-cgp/move/sources/message.move

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module axelar::message {
+    use std::string;
+    use std::string::String;
+
+    #[test_only]
+    use std::vector;
+
+    /// Message struct which can consumed only by a `Channel` object.
+    /// Does not require additional generic field to operate as linking
+    /// by `id_bytes` is more than enough.
+    ///
+    /// Consider naming this `axelar::messaging::CallApproval`.
+    struct Message has store {
+        /// ID of the message, guaranteed to be unique by Axelar.
+        msg_id: String,
```

Comment:
Should it be String? What's the size of the message? If 32 bytes, perhaps, we could use a better type for this such as `address`?